provisioning/ansible_inventory_local

```diff
@@ -0,0 +1,10 @@
+pique.nikhef.nl
+tietar.nikhef.nl
+
+vagrant ansible_connection=local
```

Comment:
IIRC, nothing. It is only useful to quickly test a new OS build or something. I think we can drop this.